src/app/feature_tracker_only.cpp

```diff
@@ -0,0 +1,95 @@
+// Author: Parth Agrawal
+#include "unistd.h"
+#include <algorithm>
+#include <vector>
+
+#include "gflags/gflags.h"
+#include "glog/logging.h"
+#include "opencv2/highgui/highgui.hpp"
+
+#include "estimator.h"
+#include "estimator_process.h"
+#include "metrics.h"
+#include "tracker.h"
+#include "loader.h"
+#include "viewer.h"
+#include "visualize.h"
+#include "graphwriter.h"
+
+// flags
+DEFINE_string(cfg, "cfg/vio.json",
```

Comment:
more nitpicking: make the default inputs "correct" for this application. So instead of vio.json, tracker_only.json.